src/orders/OrderProcessor.sol

```diff
@@ -453,6 +450,9 @@ abstract contract OrderProcessor is AccessControlDefaultAdminRules, Multicall, S
             _orders[id].feesPaid = feesPaid;
         }
 
+        // Notify order filled
+        emit OrderFill(order.recipient, index, fillAmount, receivedAmount, _orders[id].feesPaid);
```

Comment:
Keep at original location and use feesEarned. The earned fees will become paid fees later in the execution and we want the incremental fees per fill.